zebra-chain/src/primitives/zcash_primitives.rs

```diff
@@ -152,7 +152,7 @@ pub(crate) fn auth_digest(trans: &Transaction) -> AuthDigest {
 /// Return the destination address from a transparent output.
 ///
 /// Returns None if the address type is not valid or unrecognized.
-pub(crate) fn transparent_output_address(
+pub fn transparent_output_address(
```

Comment:
Was changed just to be used from a test in another crate.

Reply:
Question: Is it possible to use `Output::address()` instead?

It is already a public API:
https://github.com/ZcashFoundation/zebra/blob/44cb35c320f753853fdc0d24d7fa40a9196cf45b/zebra-chain/src/transparent.rs#L334-L339

Reply:
Thanks! https://github.com/ZcashFoundation/zebra/pull/4062/commits/1c2e3863b25281649a126c399dc7e6134af87824